src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+@import 'variables';
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $widthOfCalendar;
+  padding: 10px;
+  margin-top: 70px;
+
+  &__day {
+    display: flex;
+    border: 1px solid black;
+    width: $widthOfDay;
+    height: $heigthOfDay;
```

Comment:
There is a typo in the variable name `$heigthOfDay`. It should be `$heightOfDay` to follow the correct spelling of 'height'.